announcementsAPI.json

```diff
@@ -288,6 +296,45 @@
         }
       }
     },
+    "/announcements/{announcementId}/like": {
+      "put": {
+        "tags": [
+          "AnnouncementsEndpoints"
+        ],
+        "summary": "Adds or deletes announcement from liked announcements depending on boolean value in parameters. Requires adopter role. Gets adopter id from auth claims.",
+        "parameters": [
+          {
+            "name": "announcementId",
```

Comment:
It should rather be a path parameter, not a query one.

Reply:
Yes, you're right. I've just changed that